theorems/T000232.md

```diff
@@ -1,7 +1,9 @@
 ---
 uid: T000232
 if:
-  P000101: true
+  and:
+  - P000101: true
+  - P000129: false
```

Comment:
Same comment here: The natural hypothesis to use here should be a cardinality property ("space contains at least two points") and not the fact that the topology is indiscrete.
I did not check all the other cases, but that's probably the same thing.